calico-enterprise/networking/configuring/node-local-dns-cache.mdx

```diff
@@ -0,0 +1,103 @@
+---
+description: Improve cluster DNS performance with NodeLocal DNSCache
+---
+
+# Improve cluster DNS performance with NodeLocal DNSCache
+
+## Big picture
+
+NodeLocal DNSCache deploys `node-local-dns` pods as hostnetwork on all cluster nodes to improve DNS perfomance.
+This means the route for DNS traffic is changing and new hops are being introduced in your network when enabling this feature.
+Thus, you need appropriate policies in place to allow for this new traffic in your cluster.
+
+After deploying NodeLocal DNSCache to your {{prodname}} cluster, you need to add a policy for `kube-dns` to allow ingress
+traffic from `node-local-dns` pods. In cases where the `node-local-dns` does not have the information cached, it needs to
+be able to communicate to `kube-dns` service and thus `kube-dns` should allow this incoming traffic.
+
+In this documentation we are providing some guidance to help you define required network policies in your {{prodname}}
+cluster to allow for NodeLocal DNSCache traffic.
+
+## Value
+
+NodeLocal DNSCache is a Kubernetes feature that improves cluster DNS performance.  When you deploy NodeLocal DNSCache
+on your {{prodname}} cluster, you need to adjust your network policies accordingly to allow for the new traffic in your cluster.
+
+## Unsupported
+
+- OpenShift
+
+### OpenShift
+
+Openshift has its own DNS caching mechanism which deploys CoreDNS pods as Daemonsets. {{prodname}} is expected to function
```

Comment:
Remove the "prodname is expected to function normally in Openshift" sentence.  It implies that it doesn't function normally elsewhere, without providing any information.